### DISCUSSION TITLE
Jupyter Notebook (ipynb) is  not supported in github mobil app

### DISCUSSION THREAD

**Discussion by abhi24112** [community/community#111879] [Mobile]
Selected Topic Area:

feature 

Body:

Why is Jupiter notebook not supporting the Github mobile app and also the "raw lines" in the Jupyter files and these "raw lines" also not visible in the get up website jupyter Notebook preview.???

Github mobile:
![Screenshot_2024-03-10-15-16-49-756_com.github.android.jpg](https://github.com/community/community/assets/91802093/51ffaa3a-25f4-4ddd-af5f-1f43749c495f)

website:
in this after the "what is a python?" there is a raw cell. but this not showing in the preview or after downloading the file 

1:
![IMG_20240310_152130.jpg](https://github.com/community/community/assets/91802093/0e89b80a-a10a-4c85-888f-3f5e66201014)

2:
![IMG_20240310_152148.jpg](https://github.com/community/community/assets/91802093/c0a477e8-3807-4e34-b753-ea0f0b5d2d02)

**Comment by kydyrmyshov107** [comment]
Response #1

**Comment by benogoulart** [comment]
Hi,

This is expected behavior rather than a limitation of the GitHub Mobile app.

`raw` cells in Jupyter notebooks are intended to store content without being rendered or executed by the notebook kernel. GitHub's notebook preview focuses on displaying rendered outputs and markdown, so `raw` cells may not appear in the preview on either the website or the mobile app.

If preserving and viewing `raw` cells is important, it's generally recommended to open the notebook in a Jupyter environment (such as Jupyter Notebook, JupyterLab, or VS Code), where all cell types are displayed correctly.

If you believe `raw` cells should be visible in GitHub's notebook preview, this would make a good feature request, as it would improve consistency between GitHub's preview and the standard Jupyter experience.
